frontend/src/metabase/common/components/ColorRangeSelector/ColorRangeSelector.unit.spec.tsx

```diff
@@ -1,9 +1,14 @@
+import Color from "color";
+
 import { render, screen } from "__support__/ui";
-import { color } from "metabase/lib/colors";
+import { color as libColors } from "metabase/lib/colors";
 
 import { ColorRangeSelector } from "./ColorRangeSelector";
 import { getColorRangeLabel } from "./ColorRangeToggle";
 
+// Color components only speak hex, so we need to convert the values we use for testing to hex
+const color = (name: string) => Color(libColors(name)).hex();
```

Comment:
as a follow up, we should just export a separate object from the main color file that gives hex to all the things that need it.

Reply:
Agreed